ownCloudTests/Actions/CollaborateActionTests.swift

```diff
@@ -0,0 +1,51 @@
+//
```

Comment:
Thanks for the effort, but I think a unit test for such a small change is not necessary.